### DISCUSSION TITLE
Status of IoT hub service SDK

### DISCUSSION THREAD

**Discussion by sokolowski-daniel** [Azure/azure-iot-sdk-java#1763] [Q&A] [answered]
Dear azure-iot-sdk-java developers,

It's been a while since the IoT hub service SDK was moved to a new repository (https://github.com/Azure/azure-iot-sdk-java/discussions/1668).
Since then, no new version of the SDK has been released, even though the package depends on a vulnerable library (CVE-2023-2976 in guava).
I'm wondering what the current status of this project is and whether an update can be expected in the near future.

Thank you in advance!

**Comment by ethanann-ms** [comment, marked as answer]
We will update the vulnerable library dependency (CVE-2023-2976) and release the new version of SDK. 

Thank you

**Comment by sokolowski-daniel** [comment]
Glad to hear that, thank you
